MeshLib/IO/VtkIO/VtuInterface-impl.h

```diff
@@ -48,8 +48,10 @@ bool VtuInterface::writeVTU(std::string const &file_name, const int num_partitio
 
     vtuWriter->SetInputConnection(vtkSource->GetOutputPort());
 
-    if(_use_compressor && num_partitions == 0)
```

Comment:
@wenqing I could not figure out if removing this condition is OK or not. For the serial case `num_partitions` is 1 and now it is working as advertised. I'm not sure in which case the number of partitions would be zero.
So this might need an additional test case....